### PR TITLE
Feat memkind dax kmem nodes

### PR DESCRIPTION
**Pull request by michalbiesek**
### Description
Add memkind-dax-kmem-nodes binary to detect correct KMEM NUMA nodes
### Types of changes

- [ ] Bugfix (non-breaking change which fixes issue linked in Description above)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance)
- [ ] Other

### Checklist

- [ ] Code compiles without errors
- [ ] All tests pass locally with my changes (see TESTING section in CONTRIBUTING file)
- [ ] Created tests which will fail without the change (if possible)
- [ ] Extended the README/documentation (if necessary)
- [ ] All newly added files have proprietary license (if necessary)
- [ ] All newly added files are referenced in MANIFEST files (if necessary)

## Further comments

---
This change is [Reviewable](https://reviewable.io/reviews/memkind/memkind/251)

### CODE REVIEW

**Comment by michalbiesek** [comment]
@jschmieg 
It is ready to review - I recommend to review this commit by commit.

File: man/memkind-auto-dax-kmem-nodes.1 [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+.\" INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+.\" LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+.\" PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+.\" LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+.\" OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+.\" ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+.\"
+.TH "MEMKIND-AUTO-DAX-KMEM-NODES" 1 "2019-08-29" "Intel Corporation" "MEMKIND-AUTO-DAX-KMEM-NODES" \" -*- nroff -*-
+.SH "NAME"
+memkind-auto-dax-kmem-nodes - prints comma separated list of automatically detected persistent memory NUMA nodes.
```
Comment (jschmieg) [inline man/memkind-auto-dax-kmem-nodes.1]: a comma-separated

File: man/memkind-auto-dax-kmem-nodes.1 [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+.br
+Note: This is EXPERIMENTAL API. The functionality and the header file itself can be changed (including non-backward compatible changes), or removed.
+.SH "SYNOPSIS"
+memkind-auto-dax-kmem-nodes [-h|--help]
+.SH "DESCRIPTION"
+.BR memkind-auto-dax-kmem-nodes
+prints a comma separated list of automatically detected persistent memory NUMA nodes that can be used with the numactl --membind option.
```
Comment (jschmieg) [inline man/memkind-auto-dax-kmem-nodes.1]: comma-separated

File: man/memkind.3 [hunk 3]
```diff
@@ -860,6 +860,9 @@ Unspecified run-time error
 .TP
 .I /usr/bin/memkind-hbw-nodes
 Prints a comma separated list of high bandwidth nodes.
+.TP
+.I /usr/bin/memkind-auto-dax-kmem-nodes
+Prints a comma separated list of persistent memory NUMA nodes, which are automatically detected.
```
Comment (jschmieg) [inline man/memkind.3]: comma-separated

File: src/memkind-auto-dax-kmem-nodes.c [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+#define MAX_ARG_LEN 8
+
+static const char *const help_message =
+    "\n"
+    "NAME\n"
+    "    memkind-auto-dax-kmem-nodes - prints comma separated list of automatically detected persistent memory NUMA nodes.\n"
+    "\n"
+    "SYNOPSIS\n"
+    "    memkind-auto-dax-kmem-nodes -h | --help\n"
+    "        Print this help message.\n"
```
Comment (jschmieg) [inline src/memkind-auto-dax-kmem-nodes.c]: Prints

File: man/memkind-auto-dax-kmem-nodes.1 [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+.SH "OPTIONS"
+.BR " -h, --help"
+              Display help text and exit.
```
Comment (jschmieg) [inline man/memkind-auto-dax-kmem-nodes.1]: Displays

**Pull request merged**